py_ecc/optimized_bls12_381/optimized_curve.py

```diff
@@ -18,6 +18,12 @@
     52435875175126190479447740508185965837690552500527637822603658699938581184513
 )
 
+# Cache zero and one values for faster comparison and operations
+_FQ_ZERO = FQ.zero()
+_FQ_ONE = FQ.one()
+_FQ2_ZERO = FQ2.zero()
+_FQ2_ONE = FQ2.one()
```

Comment:
This ends up being quite minimal so I wouldn't be opposed to leaving it out for easier readability.

Reply:
I think it's about the same readability-wise, so if it gives us a little bump I'd vote to keep it